config.go

```diff
@@ -440,8 +440,18 @@ func (b *Builder) AddVolume(v string) {
 // mounted from outside of the container when a container based on an image
 // built from this container is run.
 func (b *Builder) RemoveVolume(v string) {
-	delete(b.OCIv1.Config.Volumes, v)
-	delete(b.Docker.Config.Volumes, v)
+	_, OCIv1Volume := b.OCIv1.Config.Volumes[v]
```

Comment:
Why did you add this section?  Seems like we could get a few errors when volumes only exists in OCI instead of Docker, but I am not sure this is possible.

Reply:
I was unsure whether or not to add this part, but I thought it would be good to let the user know they were trying to remove a volume that was never set in the first place. It might be redundant to check both maps though, since they are always set and unset at the same time.

Reply:
How about you check both, and if neither removes, then throw an error.